src/types/libraries.ts

```diff
@@ -0,0 +1,15 @@
+export enum Language {
```

Comment:
usually I consider an `enum` as a constant instead of a type, since it can be assigned to a variable. what do you think?

Reply:
Yeah, but we've had enums in type files before too. 🤔 I think it's a good idea to keep these with types as our type files are already segregated and not very huge.